modules/java_api/org/intel/openvino/Core.java

```diff
@@ -41,6 +41,7 @@ private static String getLibraryName(String name, String linux_ver) {
 
     // Use this method to initialize native libraries and other files like
     // plugins.xml and *.mvcmd in case of the JAR package os OpenVINO.
+    // Doesn't work for Android!
```

Comment:
Maybe we can just detect Android system and do loadLibrary inside?

Reply:
Maybe in the next PR